src/components/App.js

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
 import Display from './Display';
 import ButtonPanels from './ButtonPanel';
+import calculate from '../logic/calculate';
 
 const App = () => (
   <React.Fragment key="key">
     <Display />
-    <ButtonPanels />
+    <ButtonPanels calculate={calculate} />
```

Comment:
- [ ] Please let me ask you, why ButtonPanel is receiving calculate as a prop?

Reply:
I did it because I needed to include calculate on my app.js but if I only imported without using it, I get an ESlint error.
I'll change it so that it does not look like that and is more intuitive.